trl/trainer/sft_trainer.py

```diff
@@ -262,6 +263,129 @@ def _convert_seq_lengths_to_position_ids(batch_seq_lengths: list[list[int]]) ->
         return list(position_ids.split(example_lengths))
 
 
+@dataclass
+class DataCollatorForVisionLanguageModeling(DataCollatorMixin):
```

Comment:
Should this (and `DataCollatorForLanguageModeling`) be added to sft_trainer docs?